datafusion/src/execution/context.rs

```diff
@@ -901,14 +898,13 @@ pub struct ExecutionConfig {
     /// Should Datafusion parquet reader using the predicate to prune data
     parquet_pruning: bool,
     /// Runtime configurations such as memory threshold and local disk for spill
-    pub runtime_config: RuntimeConfig,
+    pub runtime: RuntimeConfig,
```

Comment:
Renamed to `runtime` since I think the word config is useless. Consider the usage `exec_config.runtime_config.batch_size` for example. Everything in `exec_config` is config.